src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java

```diff
@@ -91,4 +94,16 @@ public static boolean isAipJourney(AsylumCase asylumCase) {
             .read(JOURNEY_TYPE, JourneyType.class)
             .map(type -> type == AIP).orElse(false);
     }
+
+    public static Map<String, String> getAppellantPersonalisation(AsylumCase asylumCase) {
```

Comment:
This should ideally be a part of PersonalisationProvider.java. I suspect it is already there, if not then can you please add it there?